.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+
+    - name: Update Aptitude cache
+      run: sudo apt update
+
+    - name: Install MySQL server
```

Comment:
what about MariaDB instead of MySQL Server ?

Reply:
If we're not using any MariaDB-specific feature, we should more industry-standard solutions like MySQL server.